music21/stream/tools.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         stream/tools.py
+# Purpose:      Additional tools for working with streams
+#
+# Authors:      Mark Gotham
+#
+# Copyright:    Copyright © 2022 Michael Scott Asato Cuthbert
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+
+from __future__ import annotations
+
+from music21 import clef
+from music21 import environment
+from music21 import key
+from music21 import meter
+from music21 import stream
+
+environLocal = environment.Environment('stream.tools')
+
+
+# ------------------------------------------------------------------------------
+
+def removeDuplicates(thisStream: stream.Stream,
+                     classesToRemove: tuple = (meter.TimeSignature, key.KeySignature, clef.Clef),
+                     inPlace: bool = True
+                     ) -> stream.Stream:
+    '''
+    The repetition of some classes like notes is common.
+    By contrast, the explicit repetition of certain other objects like clefs
+    usually indicates an error e.g., resulting from a copy'n'paste.
+    This function serves to remove those that are likely in error and make no change.
+
+    Use the `classesToRemove` argument to specify which music21 classes to check and remove.
+    The classes currently supported are: time signatures, key signatures, and clefs.
+    The `classesToRemove` arument defaults all three.
```

Comment:
Heads up that the second typo is still here

Reply:
Thanks. Where's the face-palm emoji?